Library/Formula/apache-archiva.rb

```diff
@@ -0,0 +1,56 @@
+class ApacheArchiva < Formula
+  desc "The Build Artifact Repository Manager"
+  homepage "https://archiva.apache.org/"
+  url "https://www.apache.org/dyn/closer.cgi?path=archiva/2.2.0/binaries/apache-archiva-2.2.0-bin.tar.gz"
+  sha256 "6af7c3c47c35584f729a9c139675a01f9a9819d0cdde292552fc783284a34cfa"
+
+  depends_on :java => "1.7+"
+
+  def install
+    libexec.install Dir["*"]
+
+    bin.install_symlink libexec/"bin/archiva"
+  end
+
+  def post_install
+    (var/"archiva/logs").mkpath
+    (var/"archiva/data").mkpath
+    (var/"archiva/temp").mkpath
+
+    cp_r libexec/"conf", var/"archiva"
```

Comment:
Sorry, I've just noticed this line. Does it *have* to be copied, or can it actually be moved and still run fine?

If the latter, `(var/"archiva").install libexec/"conf"` should work - Thanks!